api/http/internal.go

```diff
@@ -34,7 +34,7 @@ func NewInternalAPI(app app.App) *InternalAPI {
 }
 
 func (api *InternalAPI) Alive(c *gin.Context) {
-	c.Status(http.StatusOK)
+	c.Status(http.StatusNoContent)
 }
 
 func (api *InternalAPI) Health(c *gin.Context) {
```

Comment:
it seems that every service follows different naming, at least on uses `HealthCheck` could we have some update in the docs of creating a new repo on a standard to follow? you can also add the mongo client, context, timeout, URL, etc stuff there :)

Reply:
For this repo I was trying to follow the same naming as the `URI*` constants. A naming convention would be nice to have for the entire project, but the problem is that we already have so many different naming for things that I don't think it's worth spending time correcting.